imglist/KR3k0030_060.txt

```diff
@@ -26,13 +26,13 @@
 060-13b00	WYG 060-13b	WYG0939/WYG0939-0479b.png
 060-14a00	WYG 060-14a	WYG0939/WYG0939-0479c.png
 060-14b00	WYG 060-14b	WYG0939/WYG0939-0479d.png
-060-15a00	WYG 060-15a	WYG0939/WYG0939-0482a.png
-060-15b00	WYG 060-15b	WYG0939/WYG0939-0482b.png
-060-16a00	WYG 060-16a	WYG0939/WYG0939-0482c.png
-060-16b00	WYG 060-16b	WYG0939/WYG0939-0482d.png
-060-17a00	WYG 060-17a	WYG0939/WYG0939-0483a.png
-060-17b00	WYG 060-17b	WYG0939/WYG0939-0483b.png
-060-18a00	WYG 060-18a	WYG0939/WYG0939-0483c.png
-060-18b00	WYG 060-18b	WYG0939/WYG0939-0483d.png
-060-19a00	WYG 060-19a	WYG0939/WYG0939-0486a.png
-060-19b00	WYG 060-19b	WYG0939/WYG0939-0486b.png
+060-15a00	WYG 060-15a	WYG0939/WYG0939-0480a.png
```

Comment:
480a to 481d is indeed missing because 文淵閣四庫全書影印本 are missing these folios. Here they are inserted as placeholders so the page mappings will be correct.